examples/llm/litellm_chat_with_loadbalancing.yaml

```diff
@@ -0,0 +1,112 @@
+# This process will create a flow where the LiteLLM agent distributes requests across multiple load balancers.
+#    Solace -> LiteLLM -> Solace
+#
+# It will subscribe to `demo/question` and expect an event with the payload:
+#
+# The input message has the following schema:
+# {
+#   "text": "<question or request as text>"
+# }
+#
+# Output is published to the topic `demo/question/response`
+# 
+# It will then send an event back to Solace with the topic: `demo/question/response`
+#
+# Dependencies:
+# pip install litellm
+#
+# required ENV variables:
+# - OPENAI_API_KEY
+# - OPENAI_API_ENDPOINT
+# - OPENAI_MODEL_NAME
+# - SOLACE_BROKER_URL
+# - SOLACE_BROKER_USERNAME
+# - SOLACE_BROKER_PASSWORD
+# - SOLACE_BROKER_VPN
```

Comment:
Please update with anthropic

Reply:
Renamed the function to litellm_chat.yaml and added the Anthropic.
Note that all LiteLLM inference and embedding components use the load balancer by the current change.